AndroidIDPTemplate/app/build.gradle.kts

```diff
@@ -6,17 +6,17 @@ plugins {
 }
 
 dependencies {
-    implementation("com.salesforce.mobilesdk:MobileSync:11.1.0")
+    implementation("com.salesforce.mobilesdk:MobileSync:11.0.1")
```

Comment:
I know in dev it does not matter since we use composite build, but we should have the latest version here still.